backend/backends/__init__.py

```
@@ -21,7 +21,7 @@
 DEFAULT_LLM_MAX_TOKENS = 512
 DEFAULT_LLM_TEMPERATURE = 0.7
 
-from ..utils.platform_detect import get_backend_type
+from ..utils.platform_detect import get_backend_type, get_supported_platforms
```

Ruff (0.15.12)

[error] Module level import not at top of file

(E402)